RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
what are the results with PU?

Unless we move all "standard" workflows to use T3 (to include variations with I1/I2, M1/M2, timing or no timing), we should first confirm that these phase2 shapes are ok on T1 in the full dyn range (well, if it helps to speedup the tests, PU140 may be enough)